src/libs/actions/connections/index.ts

```diff
@@ -14,7 +14,10 @@ import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
 type ConnectionNameExceptNetSuite = Exclude<ConnectionName, typeof CONST.POLICY.CONNECTIONS.NAME.NETSUITE>;
 
-function removePolicyConnection(policyID: string, connectionName: PolicyConnectionName) {
+function removePolicyConnection(connectionName: PolicyConnectionName, policy: Policy) {
```

Comment:
```suggestion
function removePolicyConnection(policy: Policy, connectionName: PolicyConnectionName) {
```

This would read better